modules/oe_multilingual_url_suffix/tests/src/Unit/LanguageNegotiationUrlSuffixTest.php

```diff
@@ -102,7 +112,7 @@ public function testPathSuffix(string $suffix, array $suffixes, string $expected
     ]);
 
     $request = Request::create('/foo_' . $suffix, 'GET');
-    $method = new LanguageNegotiationUrlSuffix();
+    $method = new LanguageNegotiationUrlSuffix($this->aliasManager);
```

Comment:
Why is this needed? The `LanguageNegotiationUrlSuffix` has no dependency on the alias manager nor even a constructor